src/main/java/nextstep/ladder/model/Line.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.ladder.model;
+
+import java.util.*;
+
+public class Line {
+    private List<Boolean> points = new ArrayList<>();
+    Random random = new Random();
+
+    public Line (int countOfPerson){
+       points.add(random.nextBoolean());
+       for(int i = 1;i < countOfPerson-1;i++){
```

Comment:
반복문 사용시 다음과 같이 괄호 사이에 공백을 넣어주시는게 컨벤션이오니 참고 부탁드려요 :)
또한 단항 연산자(++ 혹은 --)의 경우와 .을 제외한 모든 이항(binary) 연산자는 연산수들과는 공백으로 분리되어져야 합니다 :)

[7.4절](https://myeonguni.tistory.com/1596)
https://naver.github.io/hackday-conventions-java/#space-between-keyword-parentheses

```suggestion
       for (int i = 1; i < countOfPerson - 1; i++) {
```